Users/coala_quickstart.rst

```diff
@@ -0,0 +1,77 @@
+Using coala-quickstart to Generate a coala Configuration File
+=============================================================
+
+This document aims to make people aware of this tool called
+coala-quickstart by providing a brief introduction about its features and how
+to use it, to quickly generate coala configuration files for your projects.
+
+What is coala-quickstart?
+-------------------------
+
+coala-quickstart is a cli tool that helps users to quickly get started with
+coala by generating a ``.coafile`` tailored for their project. The ``.coafile``
+is generated based on the questions answered by the users about their project.
+
+Features
+--------
+
+coala-quickstart offers the following features:
+
+- Out-of-the-box support for projects using various popular languages such as
+  C/C++, Python, JavaScript and many more with built-in check routines.
+- Automatic detection of languages used in your project.
+- Automatatically ignoring the files mentioned in the ``.gitignore`` of your
+  project's directory.
+- Automatic identification of bears that might be relevant for your project
+  based on the languages used.
+- A clean and simple interface with a well defined flow.
+
+Installation
+------------
+
+To install the latest stable version run:
+
+::
+
+    $ pip3 install coala-quickstart
+
+To install the latest development version run:
+
+::
+
+    $ pip3 install coala-quickstart --pre
```

Comment:
since rultor died `pre` tag hasn't been updated for coala. Hardly likely it's done for coala quick start. You should remove it.